### PR TITLE
implemeted masked model (uniVerse) UI

### PR DESCRIPTION
**Pull request by khashei**
font and styles are not final

### CODE REVIEW

File: src/pages/Home/context/provider.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import reducer from './uni-verse-reducer';
+import initialState from './uni-verse-initial-state';
+import { Context } from './uni-verse-context';
```
Comment (rmanium) [inline src/pages/Home/context/provider.js:4]: What was wrong with global-context ? :D the naming is bad

File: src/pages/UniVerse/ResultContainer/error-card.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      minWidth: 275,
+      margin: 20,
+    },
+    button: {
+      margin: theme.spacing.unit,
+    },
+  })
+);
+
+const ErrorCard = props => {
```
Comment (rmanium) [inline src/pages/UniVerse/ResultContainer/error-card.jsx:18]: Generally its a good practice to destruct props right here

Comment (rmanium) [reply src/pages/UniVerse/ResultContainer/error-card.jsx:18]: const ErrorCard = ({sample, code}) => {
}

File: src/pages/UniVerse/index.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (rmanium) [inline src/pages/UniVerse/index.test.js:1]: What's up with these tests? copy paste?

File: src/pages/Home/context/provider.js (hunk 1 above)
Comment (khashei) [reply src/pages/Home/context/provider.js:4]: There will be two independent contexts one for Tak Beyt UI which is old model (Uni-Verse) and one for Sher or Chand Beyt UI which is based on GPT-2. I named this on multi-Verse.

File: src/pages/UniVerse/index.test.js (hunk 3 above)
Comment (khashei) [reply src/pages/UniVerse/index.test.js:1]: Yes removed

**Pull request merged**